### PR TITLE
fix: add missing update available and arrow icon

### PR DESCRIPTION
**Pull request by montelaidev**
## **Description**

Adds a missing `Update Available` and arrow icon in the Add Snap Account Page

## **Manual testing steps**

1. Install outdated snap (for example, https://metamask.github.io/snap-simple-keyring/0.1.4/)
2. From wallet view select accounts drop down
3. Select Add snap account
4. At snaps grid the installed snap select CTA with copy "Configure"
5. See the update CTA

### **Before**
updatesnap3

### **After**
Screenshot 2023-09-28 at 22 20 54

updatesnap2
update-snap1

## **Related issues**

Fixes https://github.com/MetaMask/accounts-planning/issues/46

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained:
  - [x] What problem this PR is solving.
  - [x] How this problem was solved.
  - [x] How reviewers can test my changes.
- [x] I’ve indicated what issue this PR is linked to: Fixes #???
- [x] I’ve included tests if applicable.
- [x] I’ve documented any added code.
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)).
- [x] I’ve properly set the pull request status:
  - [x] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by brad-decker — DISMISSED** [review]
LGTM

File: app/scripts/background.js [hunk 1]
```diff
@@ -86,6 +86,21 @@ const metamaskInternalProcessHash = {
...
+const listOfPagesToShowPopupWhenMetamaskTabIsActive = [
+  // chrome versions
+  /chrome-extension:\/\/nkbihfbeogaeaoehlefnkodbefgpgknn\/home.html#add-snap-account\//u, // main build
+  /chrome-extension:\/\/ljfoeinjpaedjfecbmggjgodbgkmjkjk\/home.html#add-snap-account\//u, // flask
+  // firfox versions
+  /moz-extension:\/\/210446d6-cf13-4eed-b4aa-1a07c0c620af\/home.html#add-snap-account\//u, // main build
+  /moz-extension:\/\/aa102033-7318-4203-bc44-44cc9198cd88\/home.html#add-snap-account\//u, // flask
+  ...(process.env.IN_TEST || process.env.METAMASK_DEBUG
+    ? [
+        /\/chrome-extension:\/\/.+\/home.html#add-snap-account\//u,
```
Comment (danroc) [inline app/scripts/background.js:98]:
```suggestion
        /\/chrome-extension:\/\/.+\/home.html#add-snap-account\//u,
```

File: app/scripts/background.js [hunk 2]
```diff
@@ -86,6 +86,21 @@ const metamaskInternalProcessHash = {
...
+        /moz-extension:\/\/.+\/home.html#add-snap-account\//u,
```
Comment (danroc) [inline app/scripts/background.js:99]:
```suggestion
        /^moz-extension:\/\/.+\/home.html#add-snap-account\//u,
```

File: app/scripts/background.js [hunk 3]
```diff
@@ -86,6 +86,21 @@ const metamaskInternalProcessHash = {
   [ENVIRONMENT_TYPE_FULLSCREEN]: true,
 };
 
+const listOfPagesToShowPopupWhenMetamaskTabIsActive = [
+  // chrome versions
+  /chrome-extension:\/\/nkbihfbeogaeaoehlefnkodbefgpgknn\/home.html#add-snap-account\//u, // main build
+  /chrome-extension:\/\/ljfoeinjpaedjfecbmggjgodbgkmjkjk\/home.html#add-snap-account\//u, // flask
```
Comment (danroc) [inline app/scripts/background.js:92]:
```suggestion
  /^chrome-extension:\/\/ljfoeinjpaedjfecbmggjgodbgkmjkjk\/home.html#add-snap-account\//u, // flask
```

File: app/scripts/background.js [hunk 4]
```diff
@@ -86,6 +86,21 @@ const metamaskInternalProcessHash = {
...
+  // firfox versions
+  /moz-extension:\/\/210446d6-cf13-4eed-b4aa-1a07c0c620af\/home.html#add-snap-account\//u, // main build
```
Comment (danroc) [inline app/scripts/background.js:94]:
```suggestion
  /^moz-extension:\/\/210446d6-cf13-4eed-b4aa-1a07c0c620af\/home.html#add-snap-account\//u, // main build
```

File: app/scripts/background.js [hunk 5]
```diff
@@ -86,6 +86,21 @@ const metamaskInternalProcessHash = {
...
+  /moz-extension:\/\/aa102033-7318-4203-bc44-44cc9198cd88\/home.html#add-snap-account\//u, // flask
```
Comment (danroc) [inline app/scripts/background.js:95]:
```suggestion
  /^moz-extension:\/\/aa102033-7318-4203-bc44-44cc9198cd88\/home.html#add-snap-account\//u, // flask
```

File: app/scripts/background.js [hunk 6]
```diff
@@ -86,6 +86,21 @@ const metamaskInternalProcessHash = {
   [ENVIRONMENT_TYPE_FULLSCREEN]: true,
 };
 
+const listOfPagesToShowPopupWhenMetamaskTabIsActive = [
+  // chrome versions
+  /chrome-extension:\/\/nkbihfbeogaeaoehlefnkodbefgpgknn\/home.html#add-snap-account\//u, // main build
+  /chrome-extension:\/\/ljfoeinjpaedjfecbmggjgodbgkmjkjk\/home.html#add-snap-account\//u, // flask
+  // firfox versions
```
Comment (danroc) [inline app/scripts/background.js:93]:
```suggestion
  // firefox versions
```

**Comment by danroc** [comment]
I left a few suggestions:
- Removed the leading `/` from Chrome's URL when in dev (it wasn't working on my machine)
- Check that the match is in the beginning of the URL

File: app/scripts/background.js (hunk 6 above)
Comment (montelaidev) [reply app/scripts/background.js]: Moved to another pr

File: app/scripts/background.js (hunk 5 above)
Comment (montelaidev) [reply app/scripts/background.js]: Moved to another pr

File: app/scripts/background.js (hunk 4 above)
Comment (montelaidev) [reply app/scripts/background.js]: Moved to another pr

File: app/scripts/background.js (hunk 3 above)
Comment (montelaidev) [reply app/scripts/background.js]: Moved to another pr

File: app/scripts/background.js [hunk 7]
```diff
@@ -86,6 +86,21 @@ const metamaskInternalProcessHash = {
   [ENVIRONMENT_TYPE_FULLSCREEN]: true,
 };
 
+const listOfPagesToShowPopupWhenMetamaskTabIsActive = [
+  // chrome versions
+  /chrome-extension:\/\/nkbihfbeogaeaoehlefnkodbefgpgknn\/home.html#add-snap-account\//u, // main build
```
[the remark this answers is not in the document]
Comment (montelaidev) [reply app/scripts/background.js]: Moved to another pr

File: app/scripts/background.js (hunk 2 above)
Comment (montelaidev) [reply app/scripts/background.js]: Moved to another pr

File: app/scripts/background.js (hunk 1 above)
Comment (montelaidev) [reply app/scripts/background.js]: Moved to another pr

**Review by danroc — APPROVED** [review]

**Review by gantunesr — COMMENTED** [review]
Confirm with the snaps team if the allowlist verifies the version to be installed

**Review by gantunesr — APPROVED** [review]

**Pull request merged**